.gitignore

```
@@ -0,0 +1,5 @@
+__pycache__
+env
```

Let’s add virtual environment patterns so Python devs have an easier time:

```suggestion
env
venv
.venv
```